### ISSUE TITLE
Coveralls makes my checks fail

### ISSUE THREAD

**Issue by edgarogh** [coverallsapp/github-action#80]
Hello,

I'm currently trying coveralls on a small project. For some reason, although the actions run without a problem and the coverage result appear correctly on the website, these two checks appear out of nowhere (I'm only using Coveralls inside the "cov" action, I've haven't configured it in any other way that would make two new icons appear) and make my PR "fail" when it is perfectly correct...

Basically, I'm wondering how I got these two icons to appear and seem to run "outside" of my actions. I'm pretty sure that I never opted in for this and if that's intentional, it's kinda invasive...
The README doesn't mention anything about this.

![image](https://user-images.githubusercontent.com/46636609/116584969-334e2280-a918-11eb-81f9-701994bc44df.png)

**Comment by edgarogh** [comment]
Unchecking this fixed it for me... Weird UX choice to configure this on coveralls.io and not in a parameter of the action.

![image](https://user-images.githubusercontent.com/46636609/116590155-b7ef6f80-a91d-11eb-929e-b34e5f8abbfa.png)
